frontend/packages/noobaa-storage-plugin/src/components/activity-card/activity-card.tsx

```diff
@@ -19,17 +19,24 @@ import {
 import { getResiliencyProgress } from '@console/ceph-storage-plugin/src/utils';
 import { DATA_RESILIENCE_QUERIES } from '../../queries';
 import './activity-card.scss';
+import { NooBaaBackingStoreModel, NooBaaBucketClassModel } from '../../models';
 
 const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
 
 const isNoobaaEventObject = (event: EventKind): boolean => {
-  const eventName: string = _.get(event, 'involvedObject.name');
-  return _.startsWith(eventName, 'noobaa');
+  const eventName: string = event?.involvedObject?.name;
+  const component: string = event?.source?.component;
+  return _.startsWith(eventName, 'noobaa') || _.startsWith(component, 'noobaa');
 };
 
 const noobaaEventsFilter = (event: EventKind): boolean => {
   const eventKind: string = _.get(event, 'involvedObject.kind');
-  if (eventKind === PodModel.kind || eventKind === StatefulSetModel.kind) {
+  if (
+    eventKind === PodModel.kind ||
+    eventKind === StatefulSetModel.kind ||
+    eventKind === NooBaaBackingStoreModel.kind ||
```

Comment:
Backing store is a Noobaa entity, `isNoobaaEventObject(event)` can be avoided for this.